po-refresh

```diff
@@ -168,19 +169,25 @@ def run(context, verbose=False, **kwargs):
             else:
                 execute("git", "checkout", "--", name)
 
-    # Create a pull request from these changes
     branch = task.branch(context, "po: Update from Fedora Weblate", pathspec="po/",
                          branch=local_branch, **kwargs)
 
-    if branch:
-        task.pull(branch, **kwargs)
-    elif local_branch:
+    if local_branch and not branch:  # We only introduced/dropped language but otherwise no updates
         if kwargs["issue"]:
             clean = "https://github.com/{0}".format(task.find_our_fork(user))
             task.comment_done(kwargs["issue"], "po-refresh", clean, local_branch, context)
 
         task.push_branch(user, local_branch)
-        task.pull("{0}:{1}".format(user, local_branch), **kwargs)
+        branch = "{0}:{1}".format(user, local_branch)
+
+    # Create a pull request from these changes
+    pull = task.pull(branch, labels=['bot', 'no-test'], run_tests=False, **kwargs)
+
+    # Trigger this pull request
+    api = github.GitHub()
+    head = pull["head"]["sha"]
+    api.post("statuses/{0}".format(head), {"state": "pending", "context": TEST_OS_DEFAULT,
+                                           "description": github.NOT_TESTED_DIRECT})
```

Comment:
Did you check that this is true for all our projects? E.g. cockpit-ostree comes to my mind, which can't run on fedora-32. Possibly we need to keep only the `no-test` bit and trigger the tests from the per-project workflows?

Reply:
good point!
I think we can do some helper function in `testmap.py` and the logic is going to be simple - for `cockpit-project/cockpit` trigger the default context, for anything else trigger the `master` branch context. WDYT?

Reply:
Updated